drivers/zfs/zfs.go

```diff
@@ -300,6 +303,9 @@ func (d *Driver) create(id, parent string, storageOpt map[string]string) error {
 	}
 	if parent == "" {
 		mountoptions := map[string]string{"mountpoint": "legacy"}
+		if d.options.mountOptions != "" {
+			mountoptions[d.options.mountOptions] = ""
+		}
```

Comment:
This looks like it wouldn't handle options of the `key=value` form (for example, `uid=1000`) correctly.

Reply:
FIxed